content/blog/posts/world-of-compilers.md

```diff
@@ -0,0 +1,756 @@
+---
+title: "The “Magical” World of Compilers, Linkers, and Loaders"
```

Comment:
FYI the nested quoting here might throw off the yaml parser

Reply:
Seems to load fine for me. 

![image](https://user-images.githubusercontent.com/80487462/205120941-a399cbed-ae26-4da9-9e5d-7adf82c51f39.png)